console/src/schematic/toolbar/Toolbar.tsx

```diff
@@ -91,6 +100,15 @@ export const Toolbar = ({ layoutKey }: ToolbarProps): ReactElement | null => {
     },
     [layoutKey, state?.editable],
   );
+  const client = Synnax.use();
+  const queryResult = useQuery({
```

Comment:
Add `client?.key` to the list of dependencies so this fires if the cluster connection changes.